src/main/java/gregicadditions/GAMaterials.java

```diff
@@ -42,17 +42,21 @@
     public static final long DISABLE_AUTOGENERATED_MIXER_RECIPE = createFlag(50);
 
     public static final long GENERATE_ROUND = createFlag(51);
+    public static final long GENERATE_DOUBLE_PLATE = createFlag(52);
+
 
     static {
         Material.MatFlags.registerMaterialFlag("GENERATE_METAL_CASING", GENERATE_METAL_CASING, IngotMaterial.class);
         Material.MatFlags.registerMaterialFlag("DISABLE_REPLICATION", DISABLE_REPLICATION, IngotMaterial.class);
         Material.MatFlags.registerMaterialFlag("DISABLE_AUTOGENERATED_MIXER_RECIPE", DISABLE_AUTOGENERATED_MIXER_RECIPE, IngotMaterial.class);
         Material.MatFlags.registerMaterialFlag("GENERATE_ROUND", GENERATE_ROUND, IngotMaterial.class);
+        Material.MatFlags.registerMaterialFlag("GENERATE_DOUBLE_PLATE", GENERATE_DOUBLE_PLATE, IngotMaterial.class);
+
     }
 
     public static long STD_METAL = GENERATE_PLATE;
-    static long EXT2_METAL = GENERATE_PLATE | GENERATE_DENSE | GENERATE_ROD | GENERATE_BOLT_SCREW | GENERATE_GEAR | GENERATE_FOIL | GENERATE_FINE_WIRE | GENERATE_LONG_ROD;
-    static long CORE_METAL = EXT2_METAL | GENERATE_RING | GENERATE_FRAME | GENERATE_ROTOR | GENERATE_SMALL_GEAR | GENERATE_DENSE;
+    static long EXT2_METAL = GENERATE_PLATE | GENERATE_DENSE | GENERATE_ROD | GENERATE_BOLT_SCREW | GENERATE_GEAR | GENERATE_FOIL | GENERATE_FINE_WIRE | GENERATE_LONG_ROD | GENERATE_DOUBLE_PLATE;
+    static long CORE_METAL = EXT2_METAL | GENERATE_RING | GENERATE_FRAME | GENERATE_ROTOR | GENERATE_SMALL_GEAR | GENERATE_DENSE | GENERATE_DOUBLE_PLATE;
```

Comment:
The flag only needs to be applied to EXT2_METAL since CORE_METAL already contains all the flags of EXT2_METAL
```suggestion
    static long CORE_METAL = EXT2_METAL | GENERATE_RING | GENERATE_FRAME | GENERATE_ROTOR | GENERATE_SMALL_GEAR | GENERATE_DENSE;
```